prov/psm3/configure.m4

```diff
@@ -131,9 +131,17 @@ ifelse('
 				      [psm3_CPPFLAGS="$psm3_CPPFLAGS -URNDV_MOD_MR"])
 			      ])
 		      ])
-		AC_CONFIG_FILES([prov/psm3/psm3/psm2_hal_inlines_i.h \
+
+		AS_IF([test "x$have_psm2_src" = "x1" && test "x$psm2_happy" = "x1" && test "x$psm2_dl" != "x1" && test "x$psm3_dl" != "x1"],[
+			AC_MSG_NOTICE([Cannot build psm3 provider if psm2 provider is built with psm2 src and neither is a dl provider])
```

Comment:
Can this be structured, so I don't need a 40" monitor to read it?  Even build scripts should try to limit lines to about 80 characters.

Reply:
Will wrap.